src/aap_eda/core/models/activation.py

```diff
@@ -56,9 +56,7 @@ class Meta:
     rulebook = models.ForeignKey(
         "Rulebook", on_delete=models.SET_NULL, null=True
     )
-    extra_var = models.ForeignKey(
-        "ExtraVar", on_delete=models.CASCADE, null=True
-    )
+    extra_var = models.TextField(null=True)
```

Comment:
check if we should use JSONField here

Reply:
Do not use JSONField here.   Keep it as text to support YAML.